modules/core/src/main/scala/higherkindness/droste/package.scala

```diff
@@ -9,13 +9,30 @@ import syntax.compose._
 
 object `package` {
 
+  /** Type Alias for `F[A] => A` */
   type Algebra[F[_], A]   = GAlgebra[F, A, A]
+
+  /** Type Alias `Coalgebra[F, A]` is a type alias for `GCoalgebra[F, A, A]`.
+    * This in turn reduces to a `A => F[A]` */
   type Coalgebra[F[_], A] = GCoalgebra[F, A, A]
 
+  /** Type Alias `AlgebraM[M, F, A] = GAlgebraM[M, F, A, A]`, i.e. a `GAlgebraM` where the
+    * inner input and output types are the same.
+    *
+    * This in turn reduces to `M[A] => F[A]`, which is similar to a `FunctionK[M, F]`, except for
+    * the fact that it is not universally quantified on the value type `A`.
+    */
   type AlgebraM[M[_], F[_], A]   = GAlgebraM[M, F, A, A]
+
+  /** Type alias `CoalgebraM[M, F, A] = GCoalgebraM[M, F, A, A]`,
+    * a `GCoalgebraM`` whose inner input and output types are the same.
+    * This in turn reduces to `A => M[F[A]]`
+    */
   type CoalgebraM[M[_], F[_], A] = GCoalgebraM[M, F, A, A]
 
+  /* Reduces to `F[(R, A)] => A` */
   type RAlgebra[R, F[_], A]   = GAlgebra[F, (R, A), A]
+  /* Reduces to `A => F[R \/ A]` */
```

Comment:
"A function `A => F[Either[R, A]]`"

Reply:
I think the first scaladoc line for pretty much all of these aliases should be structured the same.